Dockerfile

```
@@ -0,0 +1,13 @@
+FROM nimlang/nim:latest AS builder
+
+COPY src/runner.nim /test-runner/
+WORKDIR /test-runner/
+RUN nim c runner.nim
```

```suggestion
RUN nim c -d:release runner.nim
```
Probably sensible. However, it's not like the `runner` does much heavy computational work.

Note that after this change the `runner` still compiles the user's solution in debug mode.

---

Not sure. If the CI falls we would want to know why though I'm not sure if this would make a difference to the compilation or runtime errors. We wouldn't debug it in the GHA anyway

---

Yes: we should be able to reproduce CI failures locally, removing `-d:release` if necessary. And in the worst case, we can always temporarily remove the flag (even just on a branch on our forked repo).

I also have an upcoming PR that improves build reproducibility.

I think it's fine to add `-d:release`. I committed this change to this PR (see https://github.com/exercism/nim-test-runner/pull/6/commits/f7380329ce347e597eaf18d5ae196daa4f0b20c7 and its commit message).